source/lib/api.js

```diff
@@ -90,7 +91,16 @@ export const getNotifications = async maxItems => {
 };
 
 export const getNotificationCount = async () => {
-	const {headers, json: notifications} = await getNotificationResponse(1);
+	const {headers, json: notifications} = await getNotificationResponse(100);
+
+	// store notification urls
+	for (let n of notifications.values()) {
+		var pullIdParts = n.subject.url.split('/').slice(-4)
+		pullIdParts.splice(2, 1, 'pull')  // hack to get url to match the links clicked in /notifications
+		let pullId = pullIdParts.join('/')
+		console.log(`pushing to local storage: ${pullId}`)
+		localStore.set(pullId, true)
```

Comment:
Same as https://github.com/sindresorhus/notifier-for-github/pull/164#discussion_r214484824